File: selectionsort.py
def selectionSort(arrayToBeSorted):
    for i in range(len(arrayToBeSorted)):
        smallerNumber = i;
        for j in range(i+1, len(arrayToBeSorted)):
            if arrayToBeSorted[smallerNumber] > arrayToBeSorted[j]:
                smallerNumber =j;
        arrayToBeSorted[i], arrayToBeSorted[smallerNumber] = arrayToBeSorted[smallerNumber], arrayToBeSorted[i]

arrayToBeSorted = [23, 4, 1, 7, 45, 99, 22];
print(arrayToBeSorted);

selectionSort(arrayToBeSorted);

print(arrayToBeSorted);

File: selectionsort.js
var arrayToBeSorted ;
function selectionSort(arrayToBeSorted){

  for(var i =0; i < arrayToBeSorted.length; i++){
    for(var j =i+ 1; j < arrayToBeSorted.length; j++){
      if(arrayToBeSorted[i] > arrayToBeSorted[j]){
         var tempArray = arrayToBeSorted[i];;
         arrayToBeSorted[i] = arrayToBeSorted[j];
       arrayToBeSorted[j] = tempArray;
     }

    }

  }
  return arrayToBeSorted;


}
arrayToBeSorted = [23, 4, 50, 7, 45, 99, 22];
console.log(arrayToBeSorted);
selectionSort(arrayToBeSorted);
console.log(arrayToBeSorted);
